test/parallel/test-assert.js

```diff
@@ -378,7 +378,7 @@ try {
 assert.strictEqual(true, threw,
                    'a.doesNotThrow is not catching type matching errors');
 
-assert.throws(function() { assert.ifError(new Error('test error')); });
+assert.throws(function() { assert.ifError(new Error('test error')), /^([A-Za-z])\w+$/});
```

Comment:
In this line, since according to [the documentation of `assert.ifError(value)`](https://nodejs.org/api/assert.html#assert_assert_iferror_value):

> Throws value if value is truthy.

We are checking that the truthy `new Error('test error')` passed to `assert.ifError()` should be rethrown, hence the error message from the block should be `/^Error: test error$/`